travel_platform/api/izi_travel/izi_travel_client.py

```diff
@@ -1,3 +1,4 @@
+from multiprocessing.pool import ThreadPool
```

Comment:
Now main page is loaded for less than a second, but getting a list of tourist attractions still takes long (3-4 sec but instead of 6-10 sec without threads), because a lot of requests are made to get an audio for each object separately. I don't know how to get it faster, can you suggest smth?